workshops/models.py

```diff
@@ -317,6 +317,13 @@ class Event(models.Model):
     latitude = models.FloatField(null=True, blank=True)
     longitude = models.FloatField(null=True, blank=True)
 
+    REPO_REGEX = re.compile(r'https://github\.com/(?P<name>[^/]+)/'
+                            r'(?P<repo>[^/]+)/?')
+    REPO_SCHEMA = 'https://github.com/{name}/{repo}/'
+    WEBSITE_REGEX = re.compile(r'(http|https)://(?P<name>[^.]+)\.github\.'
+                               r'(io|com)/(?P<repo>[^/]+)/?')
```

Comment:
I'll change to 'https?', but last time I checked, user subdomains were served from '*.github.com' too.